jobbergate-agent/charmcraft.yaml

```diff
@@ -0,0 +1,39 @@
+name: jobbergate-agent
+title: The Jobbergate Agent life-cycle manager.
```

Comment:
```suggestion
title: The Jobbergate Agent lifecycle manager.
```

Reply:
See my comment above. I'll change it to "life cycle".

Reply:
Resolved in commit e15fece.